src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountingExtractReportServiceImpl.java

```diff
@@ -220,13 +221,13 @@ protected String buildHeaderBreakFromHeader(Object header) {
         return StringUtils.repeat(KFSConstants.DASH, header.toString().length());
     }
 
-    protected void writeErrorSubReportWithNoLineItems(String subTitle, String message) {
-        writeSubTitleForErrorSubReport(subTitle);
+    protected void writeErrorSubReportWithNoLineItems(String[] subTitles, String message) {
+        writeSubTitleForErrorSubReport(subTitles);
         getReportWriterService().writeFormattedMessageLine(message);
     }
 
     protected <T extends ConcurBatchReportLineValidationErrorItem> void writeErrorSubReport(
-            List<T> errorItems, String subTitle, Consumer<T> errorItemWriter) {
+            List<T> errorItems, String[] subTitles, Consumer<T> errorItemWriter) {
```

Comment:
Another area that could benefit from converting "subTitles" to a var-args parameter, if the parameters can be reordered feasibly.